{"relationshipsSections":[{"title":"Conforms To","type":"conformsTo","kind":"relationships","identifiers":["doc:\/\/PostgresClientORM\/10Foundation14LocalizedErrorP","doc:\/\/PostgresClientORM\/s5ErrorP","doc:\/\/PostgresClientORM\/s8SendableP"]}],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"tokens":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"TableObjectError"}],"languages":["swift"]}],"kind":"declarations"}],"schemaVersion":{"patch":0,"minor":3,"major":0},"kind":"symbol","hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM"]]},"metadata":{"role":"symbol","navigatorTitle":[{"text":"TableObjectError","kind":"identifier"}],"roleHeading":"Enumeration","title":"TableObjectError","modules":[{"name":"PostgresClientORM"}],"externalID":"s:17PostgresClientORM16TableObjectErrorO","symbolKind":"enum","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"TableObjectError"}]},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/postgresclientorm\/tableobjecterror"]}],"identifier":{"url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","interfaceLanguage":"swift"},"topicSections":[{"identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/general(_:)","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/unsupported"],"title":"Enumeration Cases"},{"title":"Instance Properties","identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription"]},{"title":"Default Implementations","identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/Error-Implementations","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/LocalizedError-Implementations"],"generated":true}],"references":{"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/LocalizedError-Implementations":{"kind":"article","title":"LocalizedError Implementations","type":"topic","url":"\/documentation\/postgresclientorm\/tableobjecterror\/localizederror-implementations","abstract":[],"role":"collectionGroup","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/LocalizedError-Implementations"},"doc://PostgresClientORM/documentation/PostgresClientORM":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","url":"\/documentation\/postgresclientorm","kind":"symbol","abstract":[],"type":"topic","title":"PostgresClientORM","role":"collection"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/errorDescription":{"role":"symbol","title":"errorDescription","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription","url":"\/documentation\/postgresclientorm\/tableobjecterror\/errordescription","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"kind":"symbol","abstract":[],"type":"topic"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"role":"symbol","kind":"symbol","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","url":"\/documentation\/postgresclientorm\/tableobjecterror","title":"TableObjectError","type":"topic","navigatorTitle":[{"text":"TableObjectError","kind":"identifier"}],"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"TableObjectError","kind":"identifier"}],"abstract":[]},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/general(_:)":{"kind":"symbol","abstract":[],"url":"\/documentation\/postgresclientorm\/tableobjecterror\/general(_:)","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":")"}],"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/general(_:)","type":"topic","title":"TableObjectError.general(_:)"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/unsupported":{"title":"TableObjectError.unsupported","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"unsupported"}],"url":"\/documentation\/postgresclientorm\/tableobjecterror\/unsupported","role":"symbol","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/unsupported","abstract":[],"kind":"symbol"},"doc://PostgresClientORM/s5ErrorP":{"type":"unresolvable","title":"Swift.Error","identifier":"doc:\/\/PostgresClientORM\/s5ErrorP"},"doc://PostgresClientORM/10Foundation14LocalizedErrorP":{"type":"unresolvable","title":"Foundation.LocalizedError","identifier":"doc:\/\/PostgresClientORM\/10Foundation14LocalizedErrorP"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/Error-Implementations":{"role":"collectionGroup","title":"Error Implementations","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/Error-Implementations","url":"\/documentation\/postgresclientorm\/tableobjecterror\/error-implementations","kind":"article","abstract":[],"type":"topic"},"doc://PostgresClientORM/s8SendableP":{"type":"unresolvable","title":"Swift.Sendable","identifier":"doc:\/\/PostgresClientORM\/s8SendableP"}}}